go-controller/pkg/libovsdb/libovsdb.go

```diff
@@ -102,6 +102,7 @@ func NewSBClientWithConfig(cfg config.OvnAuthConfig, promRegistry prometheus.Reg
 
 	// Only Monitor Required SBDB tables to reduce memory overhead
 	chassisPrivate := sbdb.ChassisPrivate{}
+	igmpGroup := sbdb.IGMPGroup{}
```

Comment:
Note: there is a cost associated to monitoring a table. This may affect our scaling numbers. ☣︎